src/disco/topo/fd_topob.c

```diff
@@ -0,0 +1,428 @@
+#include "fd_topob.h"
+
+fd_topo_t
+fd_topob_new( char const * app_name ) {
```

Comment:
`fd_topo_t` is pretty big. I think sticking with the normal `footprint/align/new/join/leave/delete` conventions would be the most clear (though it might require combining the `topo` and `topob` in a strange way), but if you don't want to do that, it should be something like `fd_topob_init` that takes a `fd_topo_t * ` as the first argument.

Reply:
Another FD pattern that might be more applicable is for `join` to return an `fd_topob_t *` (which is maybe just a typedef for `fd_topo_t *`) and then `fd_topob_fini` returns an `fd_topo_t *`.

Reply:
Then the functions would be named more like `add_*`

Reply:
Leaving this for a later change.